src/sdg/api/models.py

```diff
@@ -55,6 +57,16 @@ class Token(models.Model):
         help_text=_("Wanneer het token voor het laatst is gewijzigd."),
     )
 
+    whitelisted_ips = DynamicArrayField(
+        models.CharField(max_length=15),
+        validators=[validate_ip_adress],
+        help_text=_(
+            "De IP adressen waarvan organisaties POST, UPDATE en DELETE API calls van kan maken met deze API token."
```

Comment:
migratie klopt niet met deze help tekst. Maak even opnieuw de migraties in deze PR.